auacm/app/modules/competition_manager/views.py

```diff
@@ -1,7 +1,6 @@
 from flask import request
 from flask.ext.login import current_user, login_required
 from app import app
-from app.database import session
```

Comment:
The motivation is that it got the test database to work. To be perfectly honest, I'm not 100% clear on the details, but if you do a `from...import...`, it binds the object to a new name in the importing module's namespace. Normally that isn't a problem, but if the imported object gets modified by a third module, the changes won't be reflected.

The `session` variable is initially set to the production database in `database.py`. It's then imported here when the app gets initialized. For the test suite, the `session` object gets reset to point to the test database. Doing a `from...import...` won't catch that and will still point to the production database.